driver/config/config.go

```diff
@@ -181,7 +183,11 @@ func (c *Argon2) MarshalJSON() ([]byte, error) {
 var Argon2DefaultParallelism = uint8(runtime.NumCPU() * 2)
 
 func HookStrategyKey(key, strategy string) string {
-	return fmt.Sprintf("%s.%s.hooks", key, strategy)
+	if strategy == "global" {
```

Comment:
As discussed in the other PR - please follow what I said here: https://github.com/ory/kratos/pull/1240#discussion_r619988905

Reply:
This is only in code to differentiate between strategy and "global" hooks. In the config the "hook" property can be configured directly within the "after" property.

Reply:
Ah, I overlooked that! Thank you